docs/conf.py

```diff
@@ -50,8 +52,13 @@
 
 # General information about the project.
 project = u'nixnet'
-copyright = u'2017, National Instruments'
 author = u'National Instruments'
+copyright_year = 2018
+copyright = u'%s, %s' % (copyright_year, author)
+
+current_year = datetime.datetime.now().year
+if copyright_year != current_year:
+    warnings.warn('Copyright year is %s. Current year is %s.' % (copyright_year, current_year))
```

Comment:
Should this be a warning?
Should we just always use datetime.datetime.now().year in the copyright?